wercker.yml

```diff
@@ -8,29 +8,31 @@ box:
 # Kubernetes takes a long time to download
 no-response-timeout: 10
 
+# TODO: remove before merge
 glide-install:
   steps:
-    - script:
-        name: Install Mercurial
-        code: |
-          apk add --update mercurial
-    - script:
-      name: Fetch dependencies
-      code: |
-        glide --home $WERCKER_CACHE_DIR/.glide install -v
+  - script:
+      name: nothing to see here
+      code: echo "bye glide"
 
 test:
   steps:
   - wercker/setup-go-workspace:
       package-dir: github.com/pusher/k8s-spot-rescheduler
+  - script:
+      name: dep ensure
+      code: dep ensure -v
```

Comment:
Is it not better to run `dep ensure -v` once and copy the cache across multiple steps as we did in the glide version?